### DISCUSSION TITLE
balance/balance4-conclusion

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#219] [General]
# balance/balance4-conclusion

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/balance/balance4-conclusion.html

**Comment by eternal-heathens** [comment]
处理客户端 A 请求的后端相比处理客户端 B 请求的后端负载相差很多倍 改成 哈希算法将该客户端的流量集中到某一台服务器上，那么这台服务器可能会成为瓶颈，而其他服务器则可能处于闲置状态。会更容易理解点
